games-util/gamemode/gamemode-1.2.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit meson multilib-minimal ninja-utils user
+
+DESCRIPTION="Optimise Linux system performance on demand"
+HOMEPAGE="https://github.com/FeralInteractive/gamemode"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/FeralInteractive/gamemode.git"
+	GAMEMODE_GIT_PTR="master"
+	inherit git-r3
+else
+	GAMEMODE_GIT_PTR="${PV}"
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${GAMEMODE_GIT_PTR}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}"
+
+pkg_pretend() {
+	elog
+	elog "GameMode needs a kernel capable of SCHED_ISO to use its soft realtime"
+	elog "feature. Examples of kernels providing that are sys-kernel/ck-source"
+	elog "and sys-kernel/pf-sources."
+	elog
+	elog "Support for soft readtime is completely optional. It may provide the"
```

Comment:
readtime or realtime? ;-P

Reply:
As long as this is the only thing left, I'm happy. :-D